server/src/main/java/org/cloudfoundry/identity/uaa/provider/JdbcIdentityProviderProvisioning.java

```diff
@@ -218,15 +219,21 @@ public IdentityProvider mapRow(ResultSet rs, int rowNum) throws SQLException {
                 switch (identityProvider.getType()) {
                     case OriginKeys.SAML:
                         definition = JsonUtils.readValue(config, SamlIdentityProviderDefinition.class);
-                        Optional.ofNullable(definition).map(SamlIdentityProviderDefinition.class::cast).ifPresent(e -> e.setIdpEntityId(externId));
+                        if (isNotEmpty(externId)) {
```

Comment:
Can we rename `externId` to `externalKey` to be consistent with the column name?

Reply:
done